pkg/build/controller/strategy/util.go

```diff
@@ -113,20 +113,20 @@ func setupDockerSocket(pod *corev1.Pod) {
 
 // mountConfigMapVolume is a helper method responsible for actual mounting configMap
 // volumes into a pod.
-func mountConfigMapVolume(pod *corev1.Pod, container *corev1.Container, configMapName, mountPath, volumeSuffix string, volumeSource *corev1.VolumeSource) {
-	mountVolume(pod, container, configMapName, mountPath, volumeSuffix, policy.ConfigMap, volumeSource)
+func mountConfigMapVolume(pod *corev1.Pod, container *corev1.Container, configMapName, mountPath, volumeSuffix string, volumeSource *corev1.VolumeSource, configMapMode int32) {
+	mountVolume(pod, container, configMapName, mountPath, volumeSuffix, policy.ConfigMap, volumeSource, configMapMode)
```

Comment:
We can be opinionated here and mount the ConfigMap with the default volume mode. ConfigMap volumes must be read-only in the container.